nautobot_ssot/jobs/examples.py

```diff
@@ -436,6 +436,10 @@ def load(self):
             self.job.log_debug(message=f"Loaded {region} from remote Nautobot instance")
 
         for site_entry in self._get_api_data("api/dcim/sites/"):
+            if not site_entry["status"]:
```

Comment:
Thanks for the suggestions, but...

Why not apply the same logic as below:

```py
region_name=site_entry["region"]["name"] if site_entry["region"] else None,
```

```py
status_slug=site_entry["status"]["value"] if ite_entry["status"] else None
```

Reply:
Because `status` field is required. I could assign some default value there, but seems to me better to ignore (and log) invalid objects, WDYT?

Reply:
so, if it's not required any more in the Nautobot model, maybe we could update to make it optional?
https://github.com/nautobot/nautobot-plugin-ssot/blob/develop/nautobot_ssot/jobs/examples.py#L60

Reply:
There are these options:

- Use some default status for objects that has empty status
- Fix source data in demo.nautobot.com
- Use current solution: skip objects with empty status
- Use `save()` instead of `validated_save()`
  - not nice IMHO, but will copy all data

Reply:
I will sync with @mzbroch next week to "Fix source data in demo.nautobot.com" and to understand how this can happen at all.

Reply:
I think for this we should probably just be defaulting to Active Status. The problem is definitely in the demo data somehow allowing objects that require Status to be created without it blowing up the Job but we should have the Job handling it appropriately instead of just assuming the Status is set right.

Reply:
Updated to use "active" status as a fallback for missing status

Reply:
Thanks @snaselj! Does that make this ready for review?

Reply:
@jdrew82 
Not yet, dry run for the source job went smoothly, but without dry run, the job get's stuck. Will check it and let you know once everything works.